pipeline/runtime/common.go

```diff
@@ -27,7 +27,9 @@ import (
 )
 
 const (
-	ciNewVersionGodotEnv = "CI_NEW_VERSION_GODOTENV"
+	ciNewVersionGodotEnv     = "CI_NEW_VERSION_GODOTENV"
+	outputVariableTypeString = "STRING"
```

Comment:
Can we make this an enum instead and define this in `api.go` so the contract is clear?

Reply:
done